treeherder/model/management/commands/migrate_job_info_artifacts.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import time
+from optparse import make_option
+
+import MySQLdb
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from treeherder.model.derived import ArtifactsModel
+from treeherder.model.models import (Datasource,
+                                     Job,
+                                     JobDetail,
+                                     Repository)
+
+
+class Command(BaseCommand):
+
+    help = 'Migrate existing jobs to intermediate jobs table'
+    option_list = BaseCommand.option_list + (
+        make_option('--project',
+                    action='append',
+                    dest='project',
+                    help='Filter deletion to particular project(s)',
+                    type='string'),
+        make_option('--interval',
+                    dest='interval',
+                    help='Wait specified interval between job info migrations',
+                    type='float',
+                    default=0.0))
+
+    def handle(self, *args, **options):
+        if options['project']:
+            datasources = Datasource.objects.filter(
+                project__in=options['project'])
+        else:
+            datasources = Datasource.objects.all()
+        for ds in datasources:
+            print ds.project
+            db_options = settings.DATABASES['default'].get('OPTIONS', {})
+            db = MySQLdb.connect(
+                host=settings.DATABASES['default']['HOST'],
+                db=settings.DATABASES['default']['NAME'],
+                user=settings.DATABASES['default']['USER'],
+                passwd=settings.DATABASES['default'].get('PASSWORD') or '',
+                **db_options
+            )
+
+            repository = Repository.objects.get(name=ds.project)
+            c = db.cursor()
+            offset = 0
+            limit = 10000
+            while True:
+                c.execute("""SELECT project_specific_id from job where project_specific_id>{} and repository_id={} limit 0,{}""".format(offset, repository.id, limit))
```

Comment:
Also, I believe the `order by` isn't necessary, since we'll run this script in two passes. So much can go wrong with these migration scripts I've found, I'd prefer to keep things as simple as possible. :)